Samples/macOS-Swift/macOS-Swift/AppDelegate.swift

```diff
@@ -5,8 +5,7 @@ import Sentry
 class AppDelegate: NSObject, NSApplicationDelegate {
 
     func applicationDidFinishLaunching(_ aNotification: Notification) {
-        UserDefaults.standard.register(defaults: ["NSApplicationCrashOnExceptions": true])
```

Comment:
I changed this to using the `SentryCrashExceptionApplication` in the Info.plist file.